services/packagist/packagist-base.js

```diff
@@ -85,6 +106,43 @@ class BasePackagistService extends BaseJsonService {
   getPackageName(user, repo) {
     return `${user.toLowerCase()}/${repo.toLowerCase()}`
   }
+
+  decompressResponse(json, packageName) {
+    const versions = json.packages[packageName]
+    const expanded = []
+    let expandedVersion = null
+
+    versions.forEach(versionData => {
+      if (!expandedVersion) {
+        expandedVersion = versionData
+        expanded.push(expandedVersion)
+      }
+
+      Object.entries(versionData).forEach(([key, value]) => {
+        if (value === '__unset') {
+          delete expandedVersion[key]
+        } else {
+          expandedVersion[key] = value
+        }
+      })
+
+      expandedVersion = { ...expandedVersion }
+
+      expanded.push(expandedVersion)
+    })
+
+    return expanded
+  }
+
+  findRelease(json, versions = []) {
+    json.forEach(version => {
+      versions.push(version.version)
+    })
```

Comment:
This function is not that clearly named. How about `findLatestRelease()`?

Also, we should have one method of determining the latest (stable) version that we use consistently - we shouldn't have 2 different methods for different badges. More commentry on this on the `packagist-version.js` file..

I don't think `versions` needs to be a function param here. We never pass anything in when we call it. How about we make this just

```js
const versions = json.map(version => version.version)
```